packages/@aws-cdk/aws-amazonmq/README.md

```diff
@@ -13,6 +13,15 @@
 
 <!--END STABILITY BANNER-->
 
-```ts
-import * as amazonmq from '@aws-cdk/aws-amazonmq';
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+
+```ts nofixture
+import * as aws_amazonmq from '@aws-cdk/aws-amazonmq';
 ```
+
+There are no hand-written ([L2](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib)) constructs for this service yet. 
```

Comment:
How about adding a block here such as 

```
<!-- BEGIN CFNONLY DISCLAIMER -->
...
<!-- END CFNONLY DISCLAIMER -->
```

and then checking with pkglint that this only exists for packages whose maturity > cfn-only.

Reply:
What is the advantage of doing this? I'd prefer not to add more pkglint checks unless absolutely necessary.

Reply:
Contributing forgetting to remove this when adding the first L2 to a package. 
It'll happen more often than you think.